Automattic/pocket-casts-android
# Migrate next playing adapter from data binding

## Description


Related to: #1908 

This PR migrates the whole "next playing" screen from data binding to view binding.

## Review hints

1. I've prepared this PR in a way that it's easy to review the change commit by commit, and I advise doing so.
2. I find it helpful to check "Hide white spaces" option. It makes much more easy to review some XML changes, as indentation in the file was changed. 

![image](https://github.com/Automattic/pocket-casts-android/assets/5845095/f778447f-2a75-4c80-81c6-e2bd1695b88c)

## Testing Instructions

Please carefully smoke test the "up next" view in the player. Tap all buttons, see if text values change. Add new episodes or remove. See if reordering works and the labels (episode duration, total time remaining) are correctly updated.

## Checklist
 
#### I have tested any UI changes...
